`TextPaint` to use `TextStyle` instead of `TextPaintConfig` (#1086)
flame-engine/flame: doc/text.md

font family, etc.

Example usage:

```dart
const TextPaint textPaint = TextPaint(
  style: TextStyle(
    fontSize: 48.0,
    fontFamily: 'Awesome Font',
  ),
);
```

Note: there are several packages that contain the class `TextStyle`, make sure that you import
either `package:flutter/material.dart` or `package:flutter/painting.dart` and if you also need to
import `dart:ui` you need to import it like this (since that contains another class that is also
named `TextStyle`):

```dart
import 'dart:ui' hide TextStyle;
```

Some common properties of `TextStyle` are the following (here is the
[full list](https://api.flutter.dev/flutter/painting/TextStyle-class.html)):

 - `fontFamily`: a commonly available font, like Arial (default), or a custom font added in your
 pubspec (see [here](https://flutter.io/custom-fonts/) how to do it).
 - `fontSize`: font size, in pts (default `24.0`).
 - `height`: height of text line, as a multiple of font size (default `null`).
 - `color`: the color, as a `ui.Color` (default white).

For more information regarding colors and how to create then, see the
[Colors and the Palette](palette.md) guide.

> After the creation of the `TextPaint` object you can use its `render` method to draw some string on

Reviewer: ```suggestion
After the creation of the `TextPaint` object you can use its `render` method to draw strings on
```

Author: Haha, it was your text. :smile:

Reviewer: well, it is not guaranteed that the future me agrees with the past me

Example usage:

```dart

> TextStyle style = TextStyle(color: BasicPalette.white.color);

Reviewer: ```suggestion
final style = TextStyle(color: BasicPalette.white.color);
```

> TextPaint regular = TextPaint(style: style);

Reviewer: ```suggestion
final regular = TextPaint(style: style);
```
